graphbolt/src/csc_sampling_graph.cc

```diff
@@ -221,6 +308,11 @@ c10::intrusive_ptr<SampledSubgraph> CSCSamplingGraph::SampleNeighborsImpl(
                 // This number should be the same as the result of num_pick_fn.
                 num_picked_neighbors_per_node[i + 1] =
                     picked_neighbors_cur_thread[i - begin].size(0);
+                TORCH_CHECK(
```

Comment:
For tensor read, use raw pointer as `item()` is very slow.

Reply:
Is there any other faster way? Or maybe we can delete this check.

Reply:
raw pointer `data_ptr`.